import-data.sh

```diff
@@ -64,5 +65,5 @@ mongoimport -h $DB_HOST -d agora -u $DB_USER -p $DB_PASS --authenticationDatabas
 mongo --host $DB_HOST -u $DB_USER -p $DB_PASS --authenticationDatabase admin $WORKING_DIR/create-indexes.js
 
 pushd $TEAM_IMAGES_DIR
-ls -1r *.{jpg,jpeg} | while read x; do mongofiles -h $DB_HOST -d agora -u $DB_USER -p $DB_PASS --authenticationDatabase $DB_USER -v put $x --replace; echo $x; done
+ls -1r *.{jpg,jpeg} | while read x; do mongofiles "$DB_URI" -d agora -u $DB_USER -p $DB_PASS --authenticationDatabase $DB_USER -v put $x; echo $x; done
```

Comment:
When passing the [uri](https://www.mongodb.com/docs/database-tools/mongofiles/#std-option-mongofiles.--uri)  don't you also need to set the `--uri` flag? 

The documentation seems to indicate that you can use either the `--host` or `--uri` to make connections to the DB.  I don't think we need both.  If you are going to use URI then why not just replace all occurrences or DB_HOST with DB_URI?  or just keep DB_HOST and replace its value with a uri string value?

Reply:
1. Per the [mongofiles documentation](https://www.mongodb.com/docs/database-tools/mongofiles/#options), the --uri parameter is optional:
> Starting with version 100.0 of [mongofiles](https://www.mongodb.com/docs/database-tools/mongofiles/#mongodb-binary-bin.mongofiles), the connection string may alternatively be provided as a positional parameter, without using the --uri option:

`mongofiles mongodb://[username:password@]host1[:port1][,host2[:port2],...[,hostN[:portN]]][/[database][?options]]`

2. Mongofiles and mongoimport are different tools, and I haven't validated that mongoimport will also work correctly with the retryWrites=false option that is required for mongofiles against DocumentDB. I was trying to make the minimal required changes to get this pipeline working as we are going to be eliminating the image files from the application entirely in an upcoming release. If adding the new DB_URI parameter is problematic I can do more testing and/or revise this PR to construct the required uri string using DB_HOST though.

Reply:
ahh, i see.  I think constructing the URI string using DB_HOST is a good idea, that will make it so that you don't need to pass in that string to this script at all.

also please make sure mongofiles version >= 100.0 has been installed on the EC2 instance that's running this command.

Reply:
1. I've updated the mongofiles command to use $DB_HOST, and eliminated $DB_URI.

2. From the new dev bastion:
`mongofiles version: 100.5.2`